liiweb/views/legislation.py

```diff
@@ -93,11 +102,16 @@ def get_queryset(self):
         return super().get_queryset().filter(locality=self.locality)
 
     def get_context_data(self, **kwargs):
-        return super().get_context_data(
-            locality=self.locality,
-            locality_legislation_title="Provincial Legislation",
-            page_heading=_("%(locality)s Legislation" % {"locality": self.locality}),
-            entity_profile=self.locality.entity_profile.first(),
-            entity_profile_title=self.locality.name,
-            **kwargs,
+        context = super().get_context_data(**kwargs)
+        context.update(
+            {
+                "locality": self.locality,
+                "locality_legislation_title": "Provincial Legislation",
+                "page_heading": _(
+                    "%(locality)s Legislation" % {"locality": self.locality}
+                ),
```

Comment:
```suggestion
                "page_heading": _("%(locality)s Legislation") % {"locality": self.locality},
```

the translation must happen BEFORE the interpolation.